### PR TITLE
feat(drawio): support for SVG and PNG-based drawio files

### PR DESCRIPTION
**Pull request by cathaypacific8747**
**Changes proposed:**

- [x] Add

[Draw.io](https://github.com/hediet/vscode-drawio) supports [editing](https://github.com/hediet/vscode-drawio/blob/8da9fb55cc55825fa4d2778f79b7f534a8d77bc8/src/DrawioEditorService.ts#L170-L177) `*.drawio.png`, `*.drawio.svg`, `*.dio.svg`, `*.dio.png` files.

### CODE REVIEW

File: src/iconsManifest/supportedExtensions.ts [hunk 1]
```diff
@@ -1360,7 +1360,7 @@ export const extensions: IFileCollection = {
     },
     {
       icon: 'drawio',
-      extensions: ['drawio', 'dio'],
+      extensions: ['drawio', 'drawio.svg', 'drawio.png', 'dio', 'dio.svg', 'dio.png'],
       filenamesGlob: ['.drawio'],
       extensionsGlob: [],
```
Comment (sasial-dev) [inline src/iconsManifest/supportedExtensions.ts:1365]: Simplify this for the same result!
```suggestion
      extensions: ['drawio', 'dio'],
      filenamesGlob: ['.drawio', '.dio'],
      extensionsGlob: ['png', 'svg'],
```

**Review by sasial-dev — CHANGES_REQUESTED** [review]
Your lints (and my local tests!) failed on this one.
Make sure to simplify - and test locally!

File: src/iconsManifest/supportedExtensions.ts (hunk 1 above)
Comment (sasial-dev) [reply src/iconsManifest/supportedExtensions.ts:1365]: Test locally that the above satisfies `.drawio`, `.drawio.svg`, `.drawio.png`

**Review by robertohuertasm — APPROVED** [review]
LGTM 🚀 Thanks for your contribution ❤️

**Pull request merged**